performance_manager/lib/l0_rt_trip_updates.py

```diff
@@ -210,7 +210,7 @@ def join_gtfs_static(
             )
         timestamp_lookup[min_timestamp] = int(result[0]["timestamp"])
 
-    # pylint: disable=duplicate-code
+    # pylint: enable=duplicate-code
```

Comment:
?

Reply:
we had two `pylint: disable` tags, this last one should have re-enabled the previous `disable